src/lgar.cxx

```diff
@@ -1275,7 +1334,16 @@ extern void lgar_move_wetting_fronts(double timestep_h, double *volin_cm, int wf
 	double depth_new = wf_free_drainage->depth_cm;
 
 	// loop to adjust the depth for mass balance
-	while (fabs(mass_balance_error - tolerance) > 1.E-12) {
+  int iter = 0;
+	while (fabs(mass_balance_error - tolerance) > 1.E-10) {
+    iter++;
+    if (iter>10000000) {
```

Comment:
In its current state, this code never triggers. I used it while debugging, but kept it in because it seems good to not let the loop get to this many iterations.

Reply:
I believe the code should converge in 100s of iterations (<1000 I would guess). Is it (the number 10^6) related to avoiding an infinite loop? If so, we should use a smaller number (say, 1000-5000). If the convergence does not happen in 1000-5000 iterations, we should fix it in a better way. With 10^6, the code might converge in 100,000 iterations -and we will not know about it -- which I don't think is acceptable.

Reply:
Good point. Rewrote such that the loop terminates after 1e4 iterations (rather than the original 1e7) and puts the mass balance error into AET.